src/GMB.jl

```diff
@@ -0,0 +1,71 @@
+struct GMB{S}
+    filepath :: Vector{S}
+end
+
+function GMB(dirpath)
```

Comment:
Can basically  whole function be replaced with a glob match?
We already have a [Glob.jl](https://github.com/vtjnash/Glob.jl) dependency;
as uses in:
https://github.com/JuliaText/CorpusLoaders.jl/blob/6ca038aa101603c2b1a4e017cc2b2314e17c2028/src/WikiCorpus.jl#L32

I feel lik this is something like
`glob("data/*/*/en.tags", dirpath)`

Reply:
added